### ISSUE TITLE
All h1s should be descriptive of the topic or purpose of the page

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/aurora#267]
Review page titles:
- BagIt Profile Edit
- BagIt Profile View
- Rights Statement Edit
- Transfer Details

**Comment by helrond** [comment]
Does this refer to the `<h1>` or the `<title>` tag in the page `<head>` or both?

**Comment by HaSistrunk** [comment]
Sorry that wasn't clearer: `<h1>`. I'll edit the issue to reflect that.
